web/assets/main.scss

```diff
@@ -57,7 +53,11 @@ $moj-page-width: 1220px;
     z-index: 0;
 }
 
-.moj-filter-layout__filter {
+.autocomplete__wrapper {
+  font-family: $govuk-font-family;
+}
```

Comment:
Just moved this from further up the file so that its next to other `.autocomplete...` classes